website/templates/includes/header.html

```
@@ -433,15 +434,15 @@ <h3 class="text-sm font-semibold text-[#e74c3c] uppercase mt-6 mb-4">Site Info</
             <!-- Search Bar -->
             <div class="flex-1 max-w-xl mx-4 hidden sm:block">
                 <div class="relative flex items-center rounded-lg overflow-hidden bg-white border-2 border-red-500 hover:bg-red-50 transition-colors duration-200">
-                    <form class="w-full flex items-center"
-                          action="{% url 'search' %}"
-                          method="get">
+                    <form class="w-full flex items-center dark:bg-gray-600
```

HTMLHint (1.5.0)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)